skins/oasis/modules/ContentDisplayController.class.php

```diff
@@ -62,11 +62,11 @@ public static function onDoEditSectionLink( $skin, Title $title, $section, $tool
 	 */
 	static public function onMakeHeadline( $skin, $level, $attribs, $anchor, $text, $link, $legacyAnchor, &$ret ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This function has 8 parameters, which is greater than the 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS107)